src/FileProcessor/TypoScript/Rector/ExtbasePersistenceTypoScriptRector.php

```diff
@@ -126,6 +133,7 @@ public function getMessage(): string
 
     public function configure(array $configuration): void
     {
+        $this->configuration = $configuration;
```

Comment:
If we can access whatever is defined with `$parameters->set(Option::PATHS, [$pathToExtension]);`, I guess this would be preferrable. But for now, this is my suggestion.